ledger/participant-integration-api/src/test/suite/scala/platform/indexer/RecoveringIndexerSpec.scala

```diff
@@ -173,7 +173,7 @@ class RecoveringIndexerSpec
         }
     }
 
-    "respect restart delay" in newLoggingContext { implicit logCtx =>
+    "respect restart delay" in newLoggingContext { implicit loggingContext =>
```

Comment:
See if it makes sense to use `LoggingContext.ForTesting` here and remove a level of nesting.